src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -117,7 +119,7 @@ bool MutableObjectManager::GetSemaphores(const ObjectID &object_id,
 
 void MutableObjectManager::OpenSemaphores(const ObjectID &object_id,
                                           PlasmaObjectHeader *header) {
```

Comment:
Q: is semaphores_ protected by any of these locks?

Reply:
doesn't seem to be under any sort of locking